opennms-doc/guide-admin/src/asciidoc/text/events/sources/snmp-traps.adoc

```diff
@@ -5,8 +5,35 @@
 [[ga-events-sources-snmp-traps]]
 ==== SNMP Traps
 
-If SNMP-capable devices in the network are configured to send *traps* to {opennms-product-name}, these traps are transformed into events according to pre-configured rules. The `Trapd` service daemon, which enables {opennms-product-name} to receive SNMP traps, is enabled by default.
+If SNMP-capable devices in the network are configured to send *traps* to {opennms-product-name}, these traps are transformed into events according to pre-configured rules.
+The `Trapd` service daemon, which enables {opennms-product-name} to receive SNMP traps, is enabled by default.
 
 IMPORTANT: Disabling the `Trapd` service daemon will render {opennms-product-name} *incapable* of receiving SNMP traps.
 
-Event definitions are included with {opennms-product-name} for traps from many vendors' equipment.
\ No newline at end of file
+Event definitions are included with {opennms-product-name} for traps from many vendors' equipment.
+
+[[ga-events-sources-snmp-traps-minion]]
+===== Receiving SNMP Traps from Minion
+
+If you have _Minions_ deployed in your network it is possible to enable them as a _Syslog_ receiver.
+The _Minion_ will forward the Syslog messages to the _{opennms-product-name}_ instance where they gets processed and persisted.
+To enable _SNMP Traps_ reception on the _Minion_ follow the steps below:
+
+*Step 1: Create a Trapd configuration for the Minion*
```

Comment:
Should be done via the Karaf shell instead of the directly on the filesystem to avoid having to restart the Minion.